packages/table/components/Table.tsx

```diff
@@ -421,6 +421,11 @@ export class Table<T> extends React.PureComponent<TableProps, TableState> {
     const onStartCallback = () => {
       this.setState({ resizeIndex: columnIndex });
     };
+    const dataCy = [
```

Comment:
I can't see the data-cy for this element.

Reply:
I'm seeing it:
<img width="781" alt="Screen Shot 2019-07-19 at 10 45 48 AM" src="https://user-images.githubusercontent.com/2313998/61543962-6f014c00-aa12-11e9-9825-49e5fbb2c565.png">

Reply:
This is just for the header?

Reply:
Yes